src/App.jsx

```diff
@@ -47,16 +48,19 @@ function App() {
   return (
     <div>
       {!isLoggedIn && (
-        <>
+        <div className='main-area'>
           <h1>Please login</h1>
           <LoginForm onLogin={handleLogin} />
-        </>
+        </div>
       )}
       {isLoggedIn && (
-        <>
-          <h1>Welcome, {userData.username}</h1>
-          <button onClick={handleLogout}>Log Out</button>
-        </>
+        <div className='container-div'>
+          <MenuBar />
+          <div className='main-area'>
+
+            <h1>Welcome, {userData.username}</h1>
+            <button onClick={handleLogout}>Log Out</button>
```

Comment:
Similar to above comments, all onClick and prop functionality should be refactored to incorporate typescript syntax.